src/plugins/persistentState/test/storage.unit.js

```diff
@@ -2,13 +2,17 @@ import Storage from 'handsontable/plugins/persistentState/storage';
 
 describe('persistentState', () => {
   describe('storage', () => {
+    afterAll(() => {
```

Comment:
I've investigated a problem related to reverting an original `localStorage` object and I came to the conclusion that in the unit tests every test file is tested in a separate jsdom instance, so this line (`afterAll`), in this case, is redundant.